docs/source/common/trainer.rst

```diff
@@ -907,6 +907,9 @@ Stop training once this number of epochs is reached
     # default used by the Trainer
     trainer = Trainer(max_epochs=1000)
 
+If both ``max_epochs`` and ``max_steps`` aren't specified, ``max_epochs`` will default to ``1000``.
+To disable this default, set ``max_epochs = -1``.
```

Comment:
```suggestion
To enable infinite training, set ``max_epochs = -1`` and ``max_steps = -1``.
```

Reply:
It's not necessary to set both `max_epochs = -1` and `max_steps = -1` to disable this behavior. Are you sure I should make this change?

Reply:
why not @EricWiener? as discussed before, `max_epochs=-1` and `max_steps=n` will train for n steps not infinite, so setting both to -1 is required for infinite training.

Reply:
@awaelchli you could also set `max_epochs = -1` and leave `max_steps = None` and achieve infinite training

Reply:
```suggestion
To enable infinite training, set ``max_epochs = -1``.
```